apps/android/app/src/main/java/chat/simplex/app/views/chat/ChatInfoView.kt

```diff
@@ -63,8 +63,8 @@ fun ChatInfoView(
         setContactAlias(chat.chatInfo.apiId, it, chatModel)
       },
       openPreferences = {
-        ModalManager.shared.showModal(true) {
-          ContactPreferencesView(chatModel, chatModel.currentUser.value ?: return@showModal, contact.contactId)
+        ModalManager.shared.showCustomModal { close ->
+          ContactPreferencesView(chatModel, chatModel.currentUser.value ?: return@showCustomModal, contact.contactId, close)
```

Comment:
```suggestion
          val user = chatModel.currentUser.value
          if (user != null) {
            ContactPreferencesView(chatModel, chatModel.currentUser.value, contact.contactId, close)
          }
```

Reply:
using control-flow statements (return) in function parameters is an anti-pattern, it's ok in simple one-line expressions, where it's clear what is happening, but here it is confusing.